### ISSUE TITLE
Feature: Use chats in a folder as context for new chat

### ISSUE THREAD

**Issue by guibirow** [open-webui/open-webui#6560]
# Feature Request

When working on a large and complex topic, I split the work into multiple chats with small context and specific instructions to get the answers for each topic individually, this provide more accurate and specific results. Sometimes using different models based on their quality also helps the quality of the responses.

Once the research is complete, I include the answers manually into a knowledge base, Then I pass the knowledge base as context into a final chat to consolidate the content into a final answer, but this is a very laborours process.

**Describe the solution you'd like**

With the addition of the folder feature to group multiple chats into a single folder, it would be handy to **have an option to use chats within a folder as context for starting a new chat**, similar to how we use a collection from knowledge base.
This would make flexible to use multiple conversations from different models and then pass the entire context to a final model to consolidate into a single answer.

**Describe alternatives you've considered**

Another alternative to simplify this process, would be an option to add a response into an existing collection in the knowledge base. [This could potentially be a separate ticket, let me know what you think]

**Additional context**
<none>

**Comment by medrpa** [comment]
Hi, you’re correct about the issue or feature, but a major improvement would be to integrate an OCR tool on propmt field to perform OCR on scanned PDF documents, extracting both images and text. 

Although Tika has been added, it is not functioning as expected.

**Comment by guibirow** [comment]
The feature proposed has nothing to do with OCR, the data is already available as plain text from chat history or knowledge base.
